### PR TITLE
Common layouts frontend wip

### PR DESCRIPTION
**Pull request by barmintor**

### CODE REVIEW

File: app/assets/stylesheets/portrait/_base.scss [hunk 1]
```diff
@@ -34,6 +34,14 @@
 		}
 		.dropdown-menu {
 			background-color: $color_topnav_background;
+			li {
+				a {
+					&:hover {
+						background: $color_black;
```
Comment (barmintor) [inline app/assets/stylesheets/portrait/_base.scss:40]: I'm looking at the base palette: Should the colors here be `$color_black` and `$color_white` explicitly, or should they be related to `$color_topnav: #eee;` and `$color_topnav_background: #222;`? I guess the question is whether we want to vary this by palette or not.

File: db/schema.rb [hunk 2]
```diff
@@ -89,30 +89,7 @@
 
   add_index "sites", ["slug"], name: "index_sites_on_slug", unique: true
 
-  create_table "users", force: :cascade do |t|
```
Comment (barmintor) [inline db/schema.rb:92]: I'll fix before merge, but it will require a force push

File: app/assets/stylesheets/portrait/_base.scss (hunk 1 above)
Comment (barmintor) [reply app/assets/stylesheets/portrait/_base.scss:40]: like what if we used:
```css
&:hover: {
  background-color: brightness(86%);
  color: brightness(114%);
} 
}
```

Comment (barmintor) [reply app/assets/stylesheets/portrait/_base.scss:40]: or, alternately, just document that it's always very dark gray -> black

File: app/assets/stylesheets/palettes/_base.scss [hunk 3]
```diff
@@ -3,3 +3,185 @@ $color_disabled_background: rgba(0, 0, 0, 0.2); // disable element background
 $color_facet_panel_background: rgba(255, 255, 255, 0.3); // facet panel background
 $color_topnav: #eee;
 $color_topnav_background: #222;
+
```
Comment (barmintor) [inline app/assets/stylesheets/palettes/_base.scss:6]: Thinking these definitions should be moved to `stylesheets/layouts/_base.scss`

Comment (barmintor) [reply app/assets/stylesheets/palettes/_base.scss:6]: Or maybe into a separate `_bs4_compat.scss` and included there?

File: app/assets/stylesheets/portrait/_base.scss (hunk 1 above)
Comment (barmintor) [reply app/assets/stylesheets/portrait/_base.scss:40]: to get the contrast filter to work in a fiddle I had to inherit `color` and `background-color`

File: app/assets/stylesheets/palettes/_base.scss (hunk 3 above)
Comment (er-k) [reply app/assets/stylesheets/palettes/_base.scss:6]: a bs4 compat sheet sounds great. will do!

File: app/assets/stylesheets/portrait/_base.scss (hunk 1 above)
Comment (er-k) [reply app/assets/stylesheets/portrait/_base.scss:40]: i think i've found a solution for this case in latest wip

**Comment by barmintor** [comment]
Merging initial common layouts work for Signature Collections phase I

**Pull request merged**